.buildkite/scripts/steps/test/performance.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -uo pipefail
+
+ITERATION_COUNT=`echo "$(buildkite-agent meta-data get performance-test-iteration-count)" | xargs`
+TARGET_BRANCH=`echo "$(buildkite-agent meta-data get performance-test-target-branch)" | xargs`
+
+tput setab 2; tput setaf 0; echo "Performance test will be run at ${TARGET_BRANCH} ${ITERATION_COUNT} times"
+
+cat << EOF | buildkite-agent pipeline upload
+steps:
+  - command: "git checkout $TARGET_BRANCH && git fetch && yarn kbn bootstrap && cd x-pack && yarn install && yarn test:jest"
```

Comment:
Maybe I should replace this ? @spalger 
```suggestion
  - command: "git checkout $TARGET_BRANCH && git fetch && node scripts/functional_tests --config x-pack/test/performance/config.ts"
```

Reply:
Yeah, I feel like there is a better way to checkout the right branch... mind lending your knowledge @brianseeders?